crates/topos/src/config/mod.rs

```diff
@@ -52,12 +48,12 @@ pub(crate) trait Config: Serialize {
     fn load(home: &Path, command: Option<Self::Command>) -> Result<Self::Output, figment::Error> {
         let mut figment = Figment::new();
 
-        figment = Self::load_from_file(figment, home);
-
         if let Some(command) = command {
             figment = Self::load_from_command(figment, command);
         }
 
+        figment = Self::load_from_file(figment, home);
```

Comment:
Doesn't this change the priority of config values? Seems like now config file settings will trump command line settings? That would be surprising behaviour to me.